snorkel/classification/scorer.py

```diff
@@ -36,20 +39,28 @@ def __init__(
         self,
         metrics: Optional[List[str]] = None,
         custom_metric_funcs: Optional[Mapping[str, Callable[..., float]]] = None,
+        abstain_label: Optional[int] = -1,
     ) -> None:
 
         self.metrics: Dict[str, Callable[..., float]]
         if metrics:
             for metric in metrics:
                 if metric not in METRICS:
                     raise ValueError(f"Unrecognized metric: {metric}")
-            self.metrics = {m: partial(metric_score, metric=m) for m in metrics}
+
+            filter_dict = {"golds": [abstain_label]} if abstain_label else {}
```

Comment:
what if `abstain_label = 0`?

Reply:
would work! then `filter_labels` would occur on `gold: 0`: https://github.com/HazyResearch/snorkel/blob/redux/snorkel/analysis/utils.py#L138

Reply:
then you need `if abstain_label is None`

Reply:
good catch! fixed